test/index.spec.js

```diff
@@ -1,5 +1,5 @@
 const { expect } = require('chai')
-const validatePassword = require('../index')
+const validatePassword = require('../index2')
```

Comment:
You may want both under test including your original.
You can create a spec file for each version.